migrations/versions/0002_add_service_history.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
```

Comment:
Good point. Will do an update to set existing models version to 1 and also create a history record for each.